website/docs/cli-index.html.md

```diff
@@ -9,35 +9,46 @@ description: |-
 
 # Terraform CLI Documentation
 
-Welcome to the Terraform CLI documentation!
-
-## What's in This Section of the Docs?
-
-This section contains reference documentation for Terraform's core open source
-features, including the
-[configuration language](/docs/configuration/index.html), the
-[command-line tools](/docs/commands/index.html), and the main
-[Terraform providers](/docs/providers/index.html). Use the navigation sidebar
-to browse the various subsections.
-
-## Who is This For?
-
-The Terraform CLI docs are relevant to _all Terraform users,_ including open
-source users and Terraform Cloud users.
-
-Since these docs are reference material, they are mainly written for
-_intermediate and advanced users,_ who need to find complete and detailed
-information quickly.
+This is the documentation for Terraform CLI. It is relevant to anyone working
+with Terraform's CLI-based workflows; this includes people who use Terraform CLI
+by itself, as well as those who use Terraform CLI in conjunction with Terraform
+Cloud or Terraform Enterprise.
+
+> **Hands-on:** Try the [Terraform: Get Started](https://learn.hashicorp.com/collections/terraform/aws-get-started?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) collection on HashiCorp Learn.
+
+## What's in This Documentation
```

Comment:
Same here, it might be nice to get rid of this heading and combine the sections above and below.